ansible/inventories/bionic/group_vars/all/vars.yml

```diff
@@ -124,8 +124,8 @@ postfix_pem: "{{ default_tls_host_certificate }}"
 saml2_idp_entry: login.test.max.gov
 saml2_idp_metadata_url: "https://{{ saml2_idp_entry }}/idp/shibboleth"
 saml_idp_cert_fprint: "7A43DACB95D67FB6A804FD37C8A7DCA5B0384939"
-saml2_sp_public_key: "{{ default_tls_host_key }}"
-saml2_sp_private_pem: "{{ default_tls_host_certificate }}"
+saml2_sp_public_key: "{{ default_tls_host_certificate }}"
+saml2_sp_private_pem: "{{ default_tls_host_key }}"
```

Comment:
The naming here seems wrong with `_key` -> `_certificate` and `_pem` -> `_key`. Is this a mistake, or just really confusing naming?

Reply:
Very poor naming, and that is why I had accidentally mixed them up the first time around.

I'll rename these:
- `_public_key`-> `_public_certificate`
- `_private_pem` -> `_private_key`